package.json

```diff
@@ -15,7 +15,8 @@
   "dependencies": {
     "@actions/core": "^1.8.2",
     "@actions/github": "^4.0.0",
-    "docker-compose": "^0.23.5"
+    "docker-compose": "^0.24.0",
+    "yarn": "^1.22.19"
```

Comment:
I hope to keep using npm. May I ask you to remove yarn dependency?

Reply:
just done